docs/application/dotnet/guides/tizen.circularUI/CircleDateTimeSelector.md

```diff
@@ -0,0 +1,97 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector
+summary: CircleDateTimeSelector control guide
+---
+
+# CircleDateTimeSelector
+
+`CircleDateTimeSelector` is a view for Date Picker and Time Picker.
+You can scroll it by [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+This view is the size that covers the entire screen.
+In order to receive [Rotary event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary), it must be registered as `RotaryFocusObject` property of [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage).
+
+In the Date type, you can change value of the item by touching the item and rotating bezel to set value of the year: month: day.
+For the Time type, you can change value of the hour: minute: AM / PM.
```

Comment:
The Time display screen is also a touch screen.  You can set the value of the hour: minute: AM / PM by touch and by rotating the bezel.